src/models/EventMapper.php

```diff
@@ -453,6 +453,8 @@ public function transformResults($results, $verbose)
                             . $row['ID'] . '/comments/reported';
                         $list[$key]['reported_talk_comments_uri'] = $base . '/' . $version . '/events/'
                             . $row['ID'] . '/talk_comments/reported';
+                        $list[$key]['pending_claims_uri'] = $base . '/' . $version . '/events/'
```

Comment:
@railto - as per comment on  other PR, I'm pretty sure it's inside the can Edit conditional? (Not at pc to check )